app/lib/cds_importer/entity_mapper/base_mapper.rb

```diff
@@ -108,26 +108,22 @@ def missing_entity_filter_for(missing_entities)
           end
         end
 
-        def relative_primary_key_paths_for_secondary_node
-          primary_key_paths_for_secondary_node.each_with_object({}) do |path, acc|
-            relative_path = path.sub("#{mapping_path}.", '')
-
-            acc[relative_path] = primary_key_mapping[path]
-          end
+        def entity_composite_primary_key
+          Array.wrap(entity.primary_key)
```

Comment:
Nice, I did know about the wrap method. It should be about
`[ entity.primary_key ]` 
but more explicit

Reply:
It actually behaves slightly differently:

```ruby
thing = [:foo]
Array.wrap(thing) => [:foo]

thing = :foo
Array.wrap(thing) => [:foo]

thing = [:foo]
[thing] => [[:foo]]

thing = :foo
[thing] = [:foo]
```